packages/trader/src/AppV2/Utils/positions-utils.ts

```diff
@@ -20,6 +20,24 @@ export const filterPositions = (positions: (TPortfolioPosition | TClosedPosition
         return splittedFilter.includes('main_title' in config ? config.main_title : config.name);
     });
 };
+const encryptionConfig = {
+    Accumulators: [CONTRACT_TYPES.ACCUMULATOR],
+    Vanillas: [CONTRACT_TYPES.VANILLA.CALL, CONTRACT_TYPES.VANILLA.PUT],
+    Turbos: [CONTRACT_TYPES.TURBOS.LONG, CONTRACT_TYPES.TURBOS.SHORT],
+    Multipliers: [CONTRACT_TYPES.MULTIPLIER.DOWN, CONTRACT_TYPES.MULTIPLIER.UP],
+    'Rise/Fall': [CONTRACT_TYPES.CALL, CONTRACT_TYPES.PUT],
+    'Higher/Lower': [CONTRACT_TYPES.CALL, CONTRACT_TYPES.PUT],
+    'Touch/No touch': [CONTRACT_TYPES.TOUCH.NO_TOUCH, CONTRACT_TYPES.TOUCH.ONE_TOUCH],
+    'Matches/Differs': [CONTRACT_TYPES.MATCH_DIFF.DIFF, CONTRACT_TYPES.MATCH_DIFF.MATCH],
+    'Even/Odd': [CONTRACT_TYPES.EVEN_ODD.EVEN, CONTRACT_TYPES.EVEN_ODD.ODD],
+    'Over/Under': [CONTRACT_TYPES.OVER_UNDER.OVER, CONTRACT_TYPES.OVER_UNDER.UNDER],
+};
+
+export const encryptContractFilters = (filter: string[] | [], config: Record<string, string[]> = encryptionConfig) => {
+    if (!filter.length) return [];
+    const encryptedFilter = filter.map(option => config[option] ?? []).flat();
+    return [...new Set(encryptedFilter)];
+};
```

Comment:
```suggestion
export const getContractTypes = (filter: string[] = []) => {
    if (!filter.length) return [];
    const filteredContractTypes = filter.map(option => config[option] ?? []).flat();
    return [...new Set(filteredContractTypes)];
};
```
The naming is based on the fact that we pass an array with trade types to this function, and it returns an array of all respective contract types to send to BE. Each of them is known as  `contract_type`.
And there's no use case for a second param anywhere as I can see, right? So I suggest removing the second parameter :)